corehq/apps/cloudcare/static/cloudcare/js/form_entry/entries.js

```diff
@@ -908,23 +908,69 @@ hqDefine("cloudcare/js/form_entry/entries", function () {
         self.xformParams = function () {
             return { file: self.file() };
         };
-
+        self.formplayerProcessed = false;
```

Comment:
I'd think this doesn't need to be set both here and in `Question`. Can this line be removed? (I'd keep the `Question` one since this flag gets set in `answerQuestion` which applies to all questions, not just files.)

Reply:
oof, yes this can be removed. thanks